### PR TITLE
Happ 334 form structure

### PR DESCRIPTION
**Pull request by sunghwan-park-fw**

### CODE REVIEW

File: apps/web/src/types/Values.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 export type Values = {
```
Comment (hitankar-freshworks) [inline apps/web/src/types/Values.ts:1]: can we do small case for files?

Comment (sunghwan-park-fw) [reply apps/web/src/types/Values.ts:1]: I used pascal case for a component, class, type but camel case for a file contains multiple types.
What are our naming conventions?

Comment (hitankar-freshworks) [reply apps/web/src/types/Values.ts:1]: I think the filename could be all `kebab-case`.

**Review by hitankar-freshworks — APPROVED** [review]
Small nitpick regarding file naming but looks good otherwise.

**Pull request merged**